robust/equivalent_posynomials.py

```diff
@@ -83,13 +85,22 @@ def __init__(self, p, m, simple_model, dependent_uncertainties):
         monomials = p.chop()
         for i, mon_list in enumerate(uncertain_vars_exps_mons):
             if len(mon_list) > 1 and uncertain_vars_exps[i]:
+                # Really clunky way of creating monomials from exponents...
+                # Should streamline/speed up in the future.
                 new_no_data_posynomial = 0
+                hmap = NomialMap({HashVector(uncertain_vars_exps[i]): 1.0})
+                unitarr = [k.units**v for k, v in uncertain_vars_exps[i].items() if k.units is not None]
```

Comment:
could just be [k.units**v for k, v in uncertain_vars_exps[i].items() if k.units]

Reply:
haha thanks Ned, will do!